pytest_doctestplus/plugin.py

```diff
@@ -119,6 +123,20 @@ def pytest_addoption(parser):
     parser.addoption("--doctest-only", action="store_true",
                      help="Test only doctests. Implies usage of doctest-plus.")
 
+    parser.addoption("--doctest-plus-generate-diff",
```

Comment:
this is a bit long, could we maybe just call it `--doctest-generate-diff`?

Reply:
no opinion, TBH, I thought you might try to avoid potential clashes with doctest.

Reply:
I don't think there is such a feature upstream, but again, this was just a shed :)